songs/models.py

```diff
@@ -0,0 +1,58 @@
+from django.db import models
+from django.contrib.auth import get_user_model
+
+# Create your models here.
+
+class Artist(models.Model):
+    name = models.CharField(max_length=20)
+    
+    def __str__(self):
+        return self.name
+
+
+class Album(models.Model):
+    name = models.CharField(max_length=100)
+    album_artist = models.ForeignKey(Artist, related_name='album_artist', on_delete=models.CASCADE)
+
+    def __str__(self):
+        return self.name
+
+
+class Genre(models.Model):
+    name = models.CharField(max_length=50)
+    
+    def __str__(self):
+        return self.name
+
+
+class Playlist(models.Model):
+    name = models.CharField(max_length=50, default='Liked Songs')
+    user = models.ForeignKey(get_user_model(), related_name='playlist_user', on_delete=models.CASCADE)
+
+    def __str__(self):
+        return self.name
+
+
+class Track(models.Model):
+    title = models.CharField(max_length=250)
+    slug = models.SlugField(max_length=250)
+    created = models.DateTimeField(auto_now_add=True)
```

Comment:
Use TimeStampedModel extension to add create_at and updated_at in all the models